web/rest/platform/features/provider/notificationTemplateContent/putNotificationTemplateContent.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Update notification template contents
```

Comment:
Missing post test

Reply:
I think we can't post a default notification template under the platform section.

Reply:
Test you cannot then